CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 This file documents all notable changes to the GEOS-Chem repository starting in version 14.0.0, including all GEOS-Chem Classic and GCHP run directory updates.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [14.6.0] - 2025-03-31
+### Added
+- Added updates for compatibility with BCC.
```

Comment:
Please writing out what BCC is since not all users will know.

Reply:
@lizziel: Now fixed in [371417c](https://github.com/geoschem/geos-chem/pull/2791/commits/371417cb60f87eb154838dd5c434f750a2ae7796)